contoso-supermarket/src/app.py

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env python
-from importlib import import_module
```

Comment:
Is this change intentional?

Reply:
Yes - the linter, flake8, found that issue so I fixed it. :)